src/main/kotlin/no/skatteetaten/aurora/gorg/service/OpenShiftService.kt

```diff
@@ -0,0 +1,34 @@
+package no.skatteetaten.aurora.gorg.service
+
+import io.fabric8.openshift.client.DefaultOpenShiftClient
+import io.fabric8.openshift.client.OpenShiftClient
+import no.skatteetaten.aurora.gorg.extensions.REMOVE_AFTER_LABEL
+import no.skatteetaten.aurora.gorg.extensions.TERMINATING_PHASE
+import no.skatteetaten.aurora.gorg.extensions.applicationDeploymentsTemporary
+import no.skatteetaten.aurora.gorg.extensions.toResource
+import org.springframework.stereotype.Service
+import java.time.Instant
+
+@Service
+class OpenShiftService(
+    val client: OpenShiftClient
+) {
+
+    fun findTemporaryProjects(now: Instant): List<ProjectResource> =
```

Comment:
Add Instant.now() as the default argument?

Reply:
That could work, and would make the happy path of the code look cleaner